src/Sylius/Bundle/ShopBundle/Resources/config/app/twig_hooks/cart/index.yaml

```diff
@@ -1,93 +1,89 @@
 twig_hooks:
     hooks:
         'sylius_shop.cart.page':
+            form:
+                component: 'sylius_shop:cart:form'
+                props:
+                    cart: '@=_context.cart'
+            suggested_products:
+                component: 'sylius_shop:homepage:latest_products'
```

Comment:
Probably not in this PR, but we should move this component to the common/shared directory and adjust its name

Reply:
Yes, I agree